glusterd2/brick/types.go

```diff
@@ -33,6 +33,7 @@ type MountInfo struct {
 // Brickinfo is the static information about the brick
 type Brickinfo struct {
 	ID             uuid.UUID
+	Name           string
```

Comment:
What is stored in this variable ?

Reply:
`<volname>_s<subvol-number>_b<brick-number>` similar to subvolume name